### DISCUSSION TITLE
win7 64位 闪退，

### DISCUSSION THREAD

**Discussion by miaopin128** [rejetto/hfs#642] [Q&A]
C:\Users\Administrator>"C:\Program Files\Common Files\hfsserver\hfs.exe
Node.js is only supported on Windows 8.1, Windows Server 2012 R2, or hi
Setting the NODE_SKIP_PLATFORM_CHECK environment variable to 1 skips th
check, but Node.js might not execute correctly. Any issues encountered
unsupported platforms will not be fixed.

**Comment by rejetto** [comment]
windows 7 is not supported

**Comment by webd90kb** [comment]
For win7, you can try to use webd, quite neat and simple, https://github.com/webd90kb/webd
